modules/checking-mco-status.adoc

```diff
@@ -15,11 +15,35 @@ To see the status of the Machine Config Operator, its sub-components, and the re
 ----
 $ oc get machineconfigpool
 NAME      CONFIG                  UPDATED  UPDATING   DEGRADED  MACHINECOUNT  READYMACHINECOUNT  UPDATEDMACHINECOUNT DEGRADEDMACHINECOUNT  AGE
-master    rendered-master-dd…     True     False      False     3             3                  3                                0                     4h42m
-worker    rendered-worker-fde…    True     False      False     3             3                  3                                0                     4h42m
+master    rendered-master-dd…     True     False      False     3             3                  3                                0        4h42m
+worker    rendered-worker-fde…    True     False      False     3             3                  3                                0        4h42m
 ----
 +
+where:
++
+--
+** `UPDATED` indicates that the nodes in that MCP are updated to the current machine config.
+** `UPDATING` indicates that the nodes in that MCP are being updated to the current machine config. Nodes that are being updated are not available for scheduling.
+** `DEGRADED` indicates that the nodes in that MCP failed to be updated to the current machine config. Nodes that are degraded are not available for scheduling.
+** `MACHINECOUNT` indicates the total number of machines in that MCP.
+** `READYMACHINECOUNT` indicates the total number of machines in that MCP that are ready for scheduling.
```

Comment:
And also updated?  @yuqi-zhang recently told me:

> so the `ready` machine count there is `updated` && `ready`

which calmed me down after I'd seen:

```console
$ oc --as system:admin get machineconfigpools master
NAME     CONFIG                                             UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
master   rendered-master-7c456b1e668fb3009e41acf8023cd8a2   False     True       False      3              0                   0                     0                      2y124d
```

Although I guess on that front, it would be nice to update [the godocs][1], and then these AsciiDocs could copy-paste canonical type descriptions from the Go comments, to avoid hiding information from folks investigating in-cluster with `oc describe customresourcedefinition machineconfigpools`.

[1]: https://github.com/openshift/machine-config-operator/blob/d4f0c3103e2f7acb5708f1e82f8ab16f465bd2c2/pkg/apis/machineconfiguration.openshift.io/v1/types.go#L289-L290

Reply:
We should maybe look to change that in the future, since it is a bit un-intuitive. Also doesn't really help separate the difference between updated and ready.